core/src/Revolution/Sources/modMediaSource.php

```diff
@@ -2203,6 +2219,7 @@ protected function buildManagerImagePreview($path, $ext, $width, $height, $bases
                 'wctx' => $this->ctx->get('key'),
                 'source' => $this->get('id'),
                 't' => $timestamp,
+                'ar' => 'x'
```

Comment:
What happens when exif_read_data is not available?

Reply:
Then we will see original croped image - in this case server will not know about rotating.

Reply:
The issue I see is:
- exif_read_data does not exist. So the height and width are not swapped.
- later on the thumbnail generation is auto rotated with ar = x by what ever information.

How is this auto rotation detected in phpThumb? Can that ar property be set only if exif_read_data exists? Or better can the image be rotated directly with when $exif['Orientation'] >= 5 to avoid different behaviour there?

Reply:
```
  ar = Auto Rotate: set to "x" to use EXIF orientation
       stored by camera. Can also be set to "l" or "L"
       for landscape, or "p" or "P" for portrait. "l"
       and "P" rotate the image clockwise, "L" and "p"
       rotate the image counter-clockwise.
```

If EXIF data does not exists, height and width will not swapped and phpThumb also will not rotate image.

Reply:
https://github.com/JamesHeinrich/phpThumb/blob/master/phpthumb.class.php#L2416-L2477